inc/Remote/Api.php

```diff
@@ -166,7 +166,14 @@ private function callPlugin($pluginName, $method, $args)
         }
         $this->checkAccess($methods[$method]);
         $name = $this->getMethodName($methods, $method);
-        return call_user_func_array(array($plugin, $name), $args);
+        try {
+            set_error_handler(array($this, "argumentWarningHandler"), E_WARNING); // for PHP <7.1
+            return call_user_func_array(array($plugin, $name), $args);
+        } catch (\ArgumentCountError $th) {
```

Comment:
Will this fail on PHP <7.1 because the class `\ArgumentCountError` does not exist?

Reply:
Good question, I need to run it under PHP 5.6 to double check. According to tests it works OK.

The thing is that I don't want to provide a polyfill class for this because it may break other feature checks, and if I put it inside a `if (class_exists())` block, theoretically it should work, but Scrutinizer is still complaining that it does not exist 😞.